src/SourceBuild/content/eng/tools/BinaryToolKit/Versions.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
```

Comment:
This content should go into the eng/Versions.props file.  There should only be one Versions.props for the VMR's infra.